src/main/java/com/google/api/generator/gapic/protoparser/PatternParser.java

```diff
@@ -22,7 +22,7 @@ public class PatternParser {
 
   // This method tries to parse all named segments from pattern and sort in natual order
   // e.g. /v1beta1/{table_name=tests/*}/{routing_id=instances/*}/** -> (routing_id, table_name)
-  public static Set<String> getPattenBindings(String pattern) {
```

Comment:
Good catch! Thanks for correcting my mistake!

Reply:
Oh, I actually thought it was my typo, only now realized that it was the class you created =).